flang/test/Lower/OpenMP/omp-wsloop-collapse.f90

```diff
@@ -13,13 +13,13 @@ program wsloop_collapse
   integer :: i, j, k
   integer :: a, b, c
   integer :: x
-! FIRDialect:         %[[VAL_0:.*]] = fir.alloca i32 {bindc_name = "a", uniq_name = "_QEa"}
-! FIRDialect:         %[[VAL_1:.*]] = fir.alloca i32 {bindc_name = "b", uniq_name = "_QEb"}
-! FIRDialect:         %[[VAL_2:.*]] = fir.alloca i32 {bindc_name = "c", uniq_name = "_QEc"}
-! FIRDialect:         %[[VAL_3:.*]] = fir.alloca i32 {bindc_name = "i", uniq_name = "_QEi"}
-! FIRDialect:         %[[VAL_4:.*]] = fir.alloca i32 {bindc_name = "j", uniq_name = "_QEj"}
-! FIRDialect:         %[[VAL_5:.*]] = fir.alloca i32 {bindc_name = "k", uniq_name = "_QEk"}
-! FIRDialect:         %[[VAL_6:.*]] = fir.alloca i32 {bindc_name = "x", uniq_name = "_QEx"}
+! FIRDialect:         %[[VAL_0:.*]] = fir.alloca i32 {bindc_name = "a", uniq_name = "_QFwsloop_collapseEa"}
```

Comment:
Unfortunately, I'm not sure this is really unique. If we have two files with this content:

```fortran
program collision
  integer , save :: i
end
```
and
```fortran
subroutine collision
  integer, save :: i
end
```
it looks like we'll get two distinct variables with the same (not unique) name, `_QFcollisionEi`.